packages/components/src/components/hds/table/types.ts

```diff
@@ -19,3 +19,8 @@ export enum HdsTableThSortOrderIcons {
   ArrowDown = 'arrow-down',
   SwapVertical = 'swap-vertical',
 }
+
+export enum HdsTableScope {
+  Row = 'row',
+  Col = 'col',
+}
```

Comment:
In other places we handle enums like:
```
export enum HdsTableScopeValues {
 Row = 'row',
 Col = 'col',
}

export type HdsTableScope = `${HdsTableScopeValues}`;
```

This is so Glint doesn't throw a TS error when using a string value in the template.

Reply:
ahhh I LOVE how this approach combines the benefits of enums (clear, named constants) with the flexibility of string literal types.  I had seen glint errors in the upcoming table component migration so that solves a future issue I had. Thanks Zac!!